.github/workflows/ci-go-1.18.yaml

```diff
@@ -0,0 +1,23 @@
+on:
+  workflow_call:
+    secrets:
+      ACCESS_TOKEN:
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Go
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.18"
+      - name: Grant private modules access
+        run: |
+          git config --global url."https://ubiobot:${{ secrets.ACCESS_TOKEN }}@github.com".insteadOf "https://github.com"
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Test
+        run: |
+          go test -v ./...
```

Comment:
I'd prefer if this was just `script/test` as each project may have different pre/post test logic it may want to run, having `script/test` gives us a nice pivoting point for things like that.

Reply:
👍